src/components/Button/LinkButton.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { ComponentRegister } from "../../ComponentRegister";
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link LinkButton}
+ */
+export interface LinkButtonProps extends BaseButtonProps {
+  href: string;
```

Comment:
Unlike `Link` I think this one might make sense to have `href` as optional. Just pointing it out in case you agree.